src/components/reusable/textarea.rs

```diff
@@ -93,10 +94,12 @@ pub fn TextArea<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
                     return;
                 }
             };
-            chan2.send(ChanCmd::Typing {
-                chat_id,
-                rg: cx.props.messaging.clone(),
-            });
+            if send_typing {
```

Comment:
If the extension is disabled or not available for whatever reason wouldnt this always be false?

Reply:
We could set is `true` as default and add this option to app settings where it can also be changed.